doc/changes/latest.inc

```diff
@@ -39,7 +39,7 @@ Current (0.23.dev0)
 
 Enhancements
 ~~~~~~~~~~~~
-- Add :meth:`mne.Dipole.to_mni` for more convenient  dipole.pos to MNI conversion (:gh:`9043` **by new contributor** |Valerii Chirkov|_)
+- Add :meth:`mne.Dipole.to_mni` for more convenient  dipole.pos to MNI conversion (:gh:`9043` by |Valerii Chirkov|_)
```

Comment:
Let's keep the `**by new contributor**` in here -- it's better to have it both places I think

Reply:
Alright!

Reply:
Looks like it's not there in the current version?

Reply:
Yes, sorry, I've fixed this